assets/stylesheets/mixins/_style.scss

```diff
@@ -15,3 +15,11 @@
   overflow-scrolling: touch;
   -webkit-overflow-scrolling: touch;
 }
+
+@mixin circle {
+  border-radius: 999999px;
+}
+
+@mixin rounded($rounded) {
```

Comment:
It's less characters to write `border-radius: 10px` than `@include rounded(10px)`